<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientForm</class>
 <widget class="QWidget" name="ClientForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="UserTab"/>
   </item>
   <item>
    <widget class="QTextEdit" name="Msg"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="User"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>服务器</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Server">
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>端口</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="Port">
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>6669</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Connect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Send">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>外部端口</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="OutPort">
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>6669</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>内部端口</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="InPort">
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>6669</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
